src/components/page/page_body/page_body.tsx

```diff
@@ -86,6 +88,7 @@ export const OuiPageBody = <T extends ComponentTypes>({
     style
   );
 
+  useDeprecatedPropWarning({ props: { panelled }, version: '2.0.0' });
```

Comment:
I think we should also include `panelProps` as a deprecated prop, because it's dependent on `panelled` being true.